cli.py

```diff
@@ -38,7 +40,7 @@ async def fetch_block_command(block_number: int, rpc: str):
     inspect_db_session = get_inspect_session()
     trace_db_session = get_trace_session()
 
-    inspector = MEVInspector(rpc, inspect_db_session, trace_db_session)
+    inspector = MEVInspector(rpc, inspect_db_session, trace_db_session, False)
```

Comment:
Should support here too. Should be able to use the functions already needed for the other stuff